plumbing/api.go

```diff
@@ -48,6 +52,8 @@ type APIDeps struct {
 	MsgQueryer   *msg.Queryer
 	MsgSender    *msg.Sender
 	MsgWaiter    *msg.Waiter
+	Subscriber   *ps.Subscriber
```

Comment:
previously it had alpha ordering, this is a tossup whether to keep it

Reply:
I'll restore it with a rename in combination with other follow-ups, something like `NetPublisher/NetSubscriber`